tests/unit/compiler/venom/test_call.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+
+
+@pytest.fixture
+def market_maker(get_contract):
+    contract_code = """
+from ethereum.ercs import IERC20
+
+unused: public(uint256)
+token_address: IERC20
```

Comment:
i think we can just call this `token`, because it's an IERC20 rather than an address